projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -38,57 +49,106 @@ const PieSemiCircleChart: FC< PieSemiCircleChartProps > = ( {
 	height,
 	label,
 	note,
+	showTooltips = false,
 } ) => {
 	const providerTheme = useChartTheme();
+	const [ tooltipData, setTooltipData ] = useState< TooltipData | null >( null );
 	const centerX = width / 2;
 	const centerY = height;
+	const radius = Math.min( width, height ) / 3;
 
 	const accessors = {
 		value: d => d.value,
 		sort: ( a, b ) => a.value - b.value,
-		// Use the color property from the data object as a last resort. The theme provides colours by default.
```

Comment:
Let's keep all the comments that add extra information or are not yet implemented.